test/cipher.test.js

```diff
@@ -0,0 +1,16 @@
+import cipher from '../src/cipher';
+
+test('some large code', () => {
+  const result = cipher.encrypt("this is a test you don't suposse to read this", 1);
+  expect(result).toBe('uijt;jt;b;uftu;zpv;epo(u;tvqpttf;up;sfbe;uijt');
+});
+
+test('keeping the same case', () => {
+  const result = cipher.encrypt('PoPoTiTo', 15);
+  expect(result).toBe('EdEdIxId');
+});
+
+test('encrypting punctuation', () => {
```

Comment:
This should actually be a test that punctuations are kept not encrypted :slightly_smiling_face: